intellij-lsp/src/com/github/gtache/lsp/editor/EditorEventManager.scala

```diff
@@ -805,8 +804,10 @@ class EditorEventManager(val editor: Editor, val mouseListener: EditorMouseListe
                 LOG.warn("Empty file for " + FileUtils.sanitizeURI(loc.getUri))
               }
             }
-            if (ctrlRange != null) ctrlRange.dispose()
-            ctrlRange = null
+            invokeLater(() => {
+              if (ctrlRange != null) ctrlRange.dispose()
+              ctrlRange = null
+            })
```

Comment:
This one is already in an invokeLater call (line 788), so unnecessary.

Reply:
Removed :)